ethereum/block/error_from_block.go

```diff
@@ -26,21 +27,21 @@ func ErrorFromBlock(ctx context.Context, ec *ethclient.Client, receipt *ethtypes
 	if err != nil {
 		return fmt.Errorf("unable to determine error in mined block, %w", err)
 	}
-	txMessage, err := tx.AsMessage(ethtypes.LatestSignerForChainID(chainID), nil)
+	txMessage, err := core.TransactionToMessage(tx, ethtypes.LatestSignerForChainID(chainID), nil)
```

Comment:
This PR is upgrading `go-ethereum` from `v1.11.3` to `v1.11.5`. It looks like this change happened in the `v1.11.4` released.

Reply:
I think we were using that version for abigen - can you make sure installing abigen with this version and generating the bindings works? otherwise we need to pin the abigen version

Reply:
Yes, `abigen` builds fine and running `make bindings` generates identical binding files to what we were already using.